rasa_core/channels/slack.py

```diff
@@ -135,6 +137,36 @@ def _is_button_reply(slack_event):
     def _get_button_reply(slack_event):
         return json.loads(slack_event['payload'][0])['actions'][0]['name']
 
+    @staticmethod
+    def _sanitize_user_message(text, uids_to_remove):
+        """
+        Probably a good starting point for pre-formatting of user-provided text,
+        to make NLUs life easier in case they go funky to the power of extreme.
+
+        In the current state will just drop self-mentions of bot itself
+
+        :param text: raw message
+        :param users_to_drop: a list of users to remove from the content
+        :return text:
```

Comment:
Please check out the docstring convention used throughout Rasa Core, e.g. 
https://github.com/RasaHQ/rasa_core/blob/master/rasa_core/agent.py#L474

Reply:
thanks yes will do when issuing the next changeset

Reply:
hey @ricwo and what's the convention for Return values in docstring ? I couldn't find any example in the linked file. If None I would just:
`Returns text: a parsed and cleaned string`